cypress/features/regression/experimental/selectClassic.feature

```diff
@@ -0,0 +1,104 @@
+Feature: Experimental Select classic component
+  I want to change Select component properties

```

Comment:
`classic` needs to be added to description

Reply:
fixed